backend/src/ml_space_lambda/utils/mlspace_config.py

```diff
@@ -43,28 +43,31 @@ def get_environment_variables() -> dict:
-            "SYSTEM_TAG": os.getenv("SYSTEM_TAG", "MLSpace"),
-            "DATASETS_TABLE": os.getenv("DATASETS_TABLE", "mlspace-datasets"),
-            "PROJECTS_TABLE": os.getenv("PROJECTS_TABLE", "mlspace-projects"),
-            "PROJECT_USERS_TABLE": os.getenv("PROJECT_USERS_TABLE", "mlspace-project-users"),
-            "USERS_TABLE": os.getenv("USERS_TABLE", "mlspace-users"),
-            "RESOURCE_SCHEDULE_TABLE": os.getenv("RESOURCE_SCHEDULE_TABLE", "mlspace-resource-schedule"),
-            "RESOURCE_METADATA_TABLE": os.getenv("RESOURCE_METADATA_TABLE", "mlspace-resource-metadata"),
             "APP_CONFIGURATION_TABLE": os.getenv("APP_CONFIGURATION_TABLE", "mlspace-app-configuration"),
             "AWS_DEFAULT_REGION": os.getenv("AWS_DEFAULT_REGION", "us-iso-east-1"),
+            "BUCKET": os.getenv("BUCKET", "mlspace-data-bucket"),
             "DATA_BUCKET": os.getenv("DATA_BUCKET", "mlspace-data-bucket"),
-            "EMR_CONFIG_BUCKET": os.getenv("EMR_CONFIG_BUCKET", "mlspace-emr-config-bucket"),
-            "MANAGE_IAM_ROLES": os.getenv("MANAGE_IAM_ROLES", ""),
-            "LOG_BUCKET": os.getenv("LOG_BUCKET", "mlspace-log-bucket"),
+            "DATASETS_TABLE": os.getenv("DATASETS_TABLE", "mlspace-datasets"),
             "DYNAMO_TABLE": os.getenv("DYNAMO_TABLE", "mlspace-project"),
+            "EMR_CONFIG_BUCKET": os.getenv("EMR_CONFIG_BUCKET", "mlspace-emr-config-bucket"),
             "EMR_EC2_ROLE_NAME": os.getenv("EMR_EC2_ROLE_NAME", "EMR_EC2_DefaultRole"),
             "EMR_SERVICE_ROLE_NAME": os.getenv("EMR_SERVICE_ROLE_NAME", "EMR_DefaultRole"),
             "EMR_SECURITY_CONFIGURATION": os.getenv("EMR_SECURITY_CONFIGURATION", "MLSpace-EMR-SecurityConfig"),
             "EMR_EC2_SSH_KEY": os.getenv("EMR_EC2_SSH_KEY", ""),
+            "LOG_BUCKET": os.getenv("LOG_BUCKET", "mlspace-log-bucket"),
+            "MANAGE_IAM_ROLES": os.getenv("MANAGE_IAM_ROLES", ""),
             "NEW_USER_SUSPENSION_DEFAULT": os.getenv("NEW_USER_SUSPENSION_DEFAULT", "True"),
+            "PROJECT_USERS_TABLE": os.getenv("PROJECT_USERS_TABLE", "mlspace-project-users"),
+            "PROJECTS_TABLE": os.getenv("PROJECTS_TABLE", "mlspace-projects"),
+            "RESOURCE_METADATA_TABLE": os.getenv("RESOURCE_METADATA_TABLE", "mlspace-resource-metadata"),
+            "RESOURCE_SCHEDULE_TABLE": os.getenv("RESOURCE_SCHEDULE_TABLE", "mlspace-resource-schedule"),
+            "S3_KEY": os.getenv("S3_KEY", "notebook-params.json"),
+            "SYSTEM_TAG": os.getenv("SYSTEM_TAG", "MLSpace"),
             "TRANSLATE_DATE_ROLE_ARN": os.getenv("TRANSLATE_DATE_ROLE_ARN", ""),
+            "USERS_TABLE": os.getenv("USERS_TABLE", "mlspace-users"),
+            "ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN": os.getenv("ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN", ""),
+            "NOTEBOOK_INSTANCE_CONSTRAINT_POLICY_ARN": os.getenv("NOTEBOOK_INSTANCE_CONSTRAINT_POLICY_ARN", ""),
```

Comment:
If you are attaching this to the notebook role then you don't need this as the dynamic roles will automatically pass any policies attached to that role to the dynamic roles.

Reply:
Yeah this is leftover from when I was constraining the size of notebook instances you can create from notebooks. Since we don't allow this I removed it but forgot to remove this.